.github/workflows/build-and-test.yml

```
@@ -487,6 +487,55 @@ jobs:
             throw "Failed to uninstall fluentdwinsvc."
           }
 
+  windows-zeroconfig-sources:
+    runs-on: ubuntu-latest
+    # Required permissions
+    permissions:
+      pull-requests: read
+    # Set job outputs to values from filter step
+    outputs:
+      changed: ${{ steps.filter.outputs.changed }}
+    steps:
+    - uses: dorny/paths-filter@v2
+      id: filter
+      with:
+        filters: |
+          changed:
+            - 'internal/buildscripts/packaging/installer/install.ps1'
+            - 'internal/buildscripts/packaging/msi/**'
+            - 'pkg/signalfx-agent/bundle/**'
```

Should the tests also be included?
```suggestion
            - 'pkg/signalfx-agent/bundle/**'
            - 'tests/zeroconfig/**'
```

---

Good point @jeffreyc-splunk pending...